resources/metric-ui.mjs

```diff
@@ -136,8 +136,8 @@ function renderMetricsTable(metrics, min, max) {
             ${columns}
         </tr>`;
     }
-    return `<table class="metrics-table">
-        <thead>
+    return `<table class="metrics-table" >
+        <thead  onclick="this.classList.toggle('nowrap')" >
```

Comment:
Nit: two spaces between thead and onclick.

Reply:
the current format script "should" catch this

Reply:
DONE